quick-start.md

```diff
@@ -6,37 +6,41 @@ Here's a video that shows you all the things: https://www.youtube.com/watch?v=NX
 
 ## Prerequisites
 
-We use docker-compose and Python3. That's about it.
+- [Docker](https://www.docker.com/) - to run the DB for the backend.
+- [Python](https://www.python.org/) version 3 - to run the backend.
```

Comment:
Maybe 3.9 since that is covered here

https://github.com/Umuzi-org/Tilde/tree/develop/backend#installation